src/lib/core/crash.c

```diff
@@ -323,7 +323,7 @@ crash_report_feedback_daemon(struct crash_info *cinfo)
 	tt_uuid_to_string(&INSTANCE_UUID, uuid_buf);
 	snprintf_safe("\"server_id\":\"%s\",", uuid_buf);
 	tt_uuid_to_string(&REPLICASET_UUID, uuid_buf);
-	snprintf_safe("\"cluster_id\":\"%s\",", uuid_buf);
+	snprintf_safe("\"replicaset_id\":\"%s\",", uuid_buf);
```

Comment:
I think this should match the value in the [feedback daemon](https://github.com/tarantool/tarantool/blob/1a678a5e88b0c0ba7565d0027bf819e5c47f8999/src/box/lua/feedback_daemon.lua#L110). Should we update it as well? Or maybe leave them both as is for now? After all, it's cluster_*id*, not cluster_*uuid*.

Reply:
You are right, I didn't notice that this was not just logs, it is a JSON request. Reverted this line.

Reply:
It seems it is not a big problem to update it in https://github.com/tarantool/feedback (to accept both keys as a replicaset id). Let's file an issue if you're interested.